app/config.go

```diff
@@ -427,37 +454,44 @@ func nodeKey(address sdk.Address, password string) {
 	keys := MustGetKeybase()
 	res, err := (keys).ExportPrivateKeyObject(address, password)
 	if err != nil {
-		panic(err)
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
```suggestion
              log.Fatal(err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work